docs/_data/team.js

```diff
@@ -32,6 +32,41 @@ export const core = [
     links: [
       { icon: 'github', link: 'https://github.com/Tinoooo' },
       { icon: 'twitter', link: 'https://twitter.com/@ichbintino' },
+      { icon: 'bluesky', link: 'https://bsky.app/profile/iamtino.bsky.social' },
+    ],
+    sponsor: 'https://github.com/sponsors/Tinoooo',
+  },
+]
+export const maintainers = [
+  {
+    avatar: 'https://www.github.com/damienmontastier.png',
+    name: 'Damien Montastier',
+    title: 'Freelance Frontend Developer',
+    desc: 'A passionate creative-developer',
```

Comment:
Hey @colinscz, maybe lets remove or let empty the description for the rest so its not repeated everywhere unless @damienmontastier @Tinoooo you want to put something specific here,

Reply:
Removed it and applied your suggestion. :)